portfolio/src/main/webapp/comments-page.html

```diff
@@ -14,7 +14,12 @@ <h2> Comments </h2>
           <input type="text" name="author" required><br>
           <label for="value">Comment:</label><br>
           <input type="text" name="value" required><br>
-          <input type="submit" value="Submit!" class="transparent-button">
+          <input type="submit" name = "comment-submit" value="Submit!" class="transparent-button">
+      </form>
+      <p> How many comments do you want displayed? </p>
+      <form action="/comments" method="POST">
+          <input type="number" name="max-comments" min="0" ><br>
```

Comment:
Make this required so that an empty text box gives the sam error as the comment form.

Reply:
Good idea, done.